docs/provides.md

```diff
@@ -22,19 +22,37 @@ Example usage:
 
 ```python
 from charms.reactive import when, endpoint_from_flag
+from charms import layer
 
 @when('endpoint.aws.requested')
 def handle_requests():
     aws = endpoint_from_flag('endpoint.aws.requested')
     for request in aws.requests:
         if request.instance_tags:
-            tag_instance(request.instance_id, request.instance_tags)
+            tag_instance(
+                request.instance_id,
+                request.region,
+                request.instance_tags)
         if request.requested_elb:
```

Comment:
Should this be a generic load_balancer instead of elb?

Reply:
Yep.